src/identity/identity.ts

```diff
@@ -11,6 +11,8 @@ import {
 } from '../types';
 import { IdentityType } from '../types';
 import { isConfidentialOutput, psetToUnsignedHex, decodePset } from '../utils';
+import secp256k1, { ZKP } from '@vulpemventures/secp256k1-zkp';
```

Comment:
No the acrual interface type is called ZKPInterface and can be found in liquidjs. This lib should never be imported outside tests